libs/pageserver_api/src/config.rs

```diff
@@ -183,6 +183,7 @@ pub struct ConfigToml {
     pub enable_tls_page_service_api: bool,
     pub dev_mode: bool,
     pub timeline_import_config: TimelineImportConfig,
+    pub basebackup_cache_config: BasebackupCacheConfig,
```

Comment:
Let's make this optional and not spawn the new task when it's not specified.
This way it should have no impact on prod.

Reply:
Done